<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariableTabContent</class>
 <widget class="QWidget" name="VariableTabContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="GUI::L_System::VariablesWidget" name="declaredVariables">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>814</width>
        <height>243</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="axiomLabel">
       <property name="text">
        <string>Axiom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="axiomComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loadModelButton">
       <property name="text">
        <string>Load New Model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addVariableButton">
       <property name="text">
        <string>Add Variable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteVariableButtons">
       <property name="text">
        <string>Delete SelectedVariables</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUI::L_System::VariablesWidget</class>
   <extends>QWidget</extends>
   <header>GUI/L_System/Variables/VariablesWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
